src/main/java/com/ctc/wstx/util/StringVector.java

```diff
@@ -19,6 +19,10 @@ public final class StringVector
      */
 
     public StringVector(int initialCount) {
+        if (initialCount == 0) {
```

Comment:
How about checking for `<= 0` as negative ones not ok either (will fail few lines down anyway but still)?